README.md

```diff
@@ -182,12 +180,12 @@ official [third party page][3rd-party].
 
 ## Installation from Docker image
 
-Binaries are also available as a Docker image. To find out more, check out the blog post [Your central log server in Docker](https://syslog-ng.com/blog/central-log-server-docker/).
+Binaries are also available as a Docker image. To find out more, check out the blog post, [Your central log server in Docker](https://syslog-ng.com/blog/central-log-server-docker/).
 
 ## Documentation
 
-The documentation of the latest released version of syslog-ng Open Source Edition is available [here](https://www.balabit.com/sites/default/files/documents/syslog-ng-ose-latest-guides/en/syslog-ng-ose-guide-admin/html/index.html). For earlier versions, see the [syslog-ng Documentation Page](https://syslog-ng.com/documentation). For ancient versions, see the [Balabit Documentation Archive](https://my.balabit.com/downloads/archived_documents).
+The documentation of the latest released version of syslog-ng Open Source Edition is available [here](https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.20/administration-guide). For earlier versions, see the syslog-ng [Documentation Page](https://www.syslog-ng.com/technical-documents).
 
 ## Contributing
 
-If you want to modify the source of syslog-ng, for example, to correct a bug or develop a new module, the [syslog-ng gitbook](https://syslog-ng.gitbooks.io/getting-started/content/) helps you to take the first steps with the code base.
+If you would like to contribute to syslog-ng, to fix a bug or create a new module for example, the [syslog-ng gitbook](https://syslog-ng.gitbooks.io/getting-started/content/) helps you take the first steps to working with the code base.
```

Comment:
I would leave out `for example`.

Reply:
I included `for example` because the original sentence had it. And I think the original had it because there are ways to contribute other than bugs or modules (like implementing a new feature, or working on docs as I am here :)).

I agree that getting rid of it would make the sentence more streamlined though. Would you like me to remove it?

Reply:
Please do, but keep in mind I am not somebody who can give proper grammar advice.